pkg/data/data_store.go

```diff
@@ -5,15 +5,16 @@ import (
 
 	. "github.com/CESARBR/knot-cloud-storage/pkg/entities"
 	"github.com/CESARBR/knot-cloud-storage/pkg/logging"
-	"gopkg.in/mgo.v2"
-	"gopkg.in/mgo.v2/bson"
+	"github.com/globalsign/mgo"
```

Comment:
It's important to notice this change should be grouped in the commit that updates `mgo` package in the `data/mongo.go` file too. The reason is that with only one commit the code remains cohesive and avoids breaking in an unexpected way.

Reply:
Done!